crates/ruff_linter/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -687,11 +715,34 @@ fn handle_single_name(checker: &mut Checker, expr: &Expr, value: &Expr) {
         expr.range(),
     );
 
-    let node = value.clone();
-    diagnostic.set_fix(Fix::safe_edit(Edit::range_replacement(
-        checker.generator().expr(&node),
-        expr.range(),
-    )));
+    diagnostic.set_fix(Fix::unsafe_edits(
```

Comment:
Can we extend the rule documentation with when the fix is unsafe? That would also help me understand why we changed the fix to unsafe :) If it's because of possible comments, then we should change the fix to use `checker.comment_ranges().has_comments(range)` to test if the edited range contains comments and only set the fix to unsafe if that's the case.

Reply:
It's because of possible comments. I'll use has_comments to determine safety